backends/tfhe-cuda-backend/cuda/include/integer/integer.h

```diff
@@ -48,6 +48,38 @@ typedef struct {
   uint32_t lwe_dimension;
 } CudaRadixCiphertextFFI;
 
+typedef struct {
+  // divisor-related attributes
+  uint64_t const *divisor_has_at_least_one_set;
+  uint64_t const *decomposed_divisor;
+  uint32_t const num_scalars_divisor;
+  uint32_t const active_bits_divisor;
+  uint32_t const ilog2_divisor;
+
+  bool const is_divisor_zero;
+  bool const is_abs_divisor_one;
+  bool const is_divisor_negative;
+  bool const is_divisor_pow2;
+  bool const is_divisor_wider_than_numerator;
+
+  // multiplier-related attributes
```

Comment:
multiplier -> chosen-multiplier

You could also edit the comment to explain that this multiplier is used in scalar_mul_high for scalar division?